i18n/config/crowdin.yml

```diff
@@ -43,6 +43,40 @@ files: [
   #
   "update_option" : "update_without_changes",
 
+  # Map crowdin's language code to Django
+  "languages_mapping": {
+    "locale": {
+      "es-MX": "es_MX",
+      "it": "it",
+      "th": "th",
+    }
+  }
+ },
+ {
+  #
+  # Source files filter
+  # e.g. "/resources/en/*.json"
+  #
+  "source" : "locale/en/**/*.po",
+
+  #
+  # where translations live
+  # e.g. "/resources/%two_letters_code%/%original_file_name%"
+  #
+  "translation" : "/locale/%locale_with_underscore%/LC_MESSAGES/django.po",
```

Comment:
Can we override the [`LOCALE_PATHS`](https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-LOCALE_PATHS) setting instead of creating a separate `locale` directory?

Reply:
Sort of. I can't find a way to get makemessages to send the gathered strings to a custom directory. I can get Django to find the translations in a translations/ directory, but the format the language codes need to be in is different than the format for the content translations (es_MX vs es-mx). I've made the change to put translations in a translations/ directory, lmk what you think.

We can also do follow-up work to try to standardize the locale format. The underscore format is enforced by the [Django translation system](https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#how-django-discovers-translations) but I'm not sure if the dashed format is required for content translation to work.